CHANGELOG.md

```diff
@@ -34,6 +35,105 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
```

Comment:
@robscott Do you want to mention the bump to go 1.22 and k8s 1.30?

Reply:
Good question, that will depend on the resolution to https://github.com/kubernetes-sigs/gateway-api/pull/2988#issuecomment-2065320460, will leave this open until that path becomes more clear.